### PR TITLE
[ADD] pricelist_configurator_by_bom module v.7.0

### PR DESCRIPTION
**Pull request by tafaRU**
Moving: https://code.launchpad.net/~agilebg/openerp-product-attributes/adding_pricelist_configurator_by_bom_7/+merge/203026

### CODE REVIEW

File: pricelist_configurator_by_bom/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+""",
+    'author': 'Agile Business Group',
+    'website': 'http://www.agilebg.com/',
+    'depends': ['product', 'sale', 'mrp'],
+    'init_xml': [],
+    'data': [
+            'pricelist_configurator_by_bom_view.xml',
+    ],
+    'update_xml': [
+        'security/ir.model.access.csv'
```
Comment (bwrsandman) [inline pricelist_configurator_by_bom/__openerp__.py:53]: This should go into `data`, `update_xml` is deprecated.

File: pricelist_configurator_by_bom/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    ],
+    'test': [
+        'test/pricelist_configurator_by_bom.yml'
+    ],
+    'demo_xml': [],
```
Comment (bwrsandman) [inline pricelist_configurator_by_bom/__openerp__.py]: `demo_xml` is deprecated

**Comment by tafaRU** [comment]
@bwrsandman just made the changes as you suggested. Thank you.

**Comment by bwrsandman** [comment]
Tests aren't failing because of any of your code.

:+1:

**Comment by eLBati** [comment]
:+1:

**Pull request merged**